packages/jest/src/index.ts

```diff
@@ -13,3 +13,26 @@ export {
 } from '@jest/core';
 
 export {run} from 'jest-cli';
+
+declare global {
+  export const beforeEach: typeof import('@jest/globals')['beforeEach'];
```

Comment:
`export` is not needed, because `global` namespace is always public. I added `export` just as a note future reader that these are being exposed here.